packages/react-divider/src/components/Divider/useDividerStyles.ts

```diff
@@ -0,0 +1,428 @@
+import { makeStyles, ax } from '@fluentui/react-make-styles';
+import { DividerState } from './Divider.types';
+
+/**
+ * Styles for the root slot
+ */
+export const useRootStyles = (state: DividerState): string => {
+  const result = makeStyles<DividerState>([
+    /* ROOT */
+    [
+      null,
+      tokens => ({
+        /* CSS Vars */
+        '--divider-borderMargin': '12px',
+        '--divider-flexDirection': 'row',
+        '--divider-fontColor': tokens.alias.color.neutral.neutralForeground2,
+        '--divider-fontFamily': 'Segoe UI',
+        '--divider-fontSize': '12px',
+        '--divider-fontWeight': '400',
+        '--divider-lineHeight': '17px',
+        '--divider-borderSize': '1px',
+        '--divider-borderStyle': 'solid',
```

Comment:
Not that relevant for this PR but I think this will likely change once we have the component tokens story figured out.

Reply:
I agree, these are the defaults as per design document, once the tokens story is figured out, we will need to make the appropriate changes